temporal-sdk/src/main/java/io/temporal/client/WorkflowOptions.java

```diff
@@ -112,6 +110,8 @@ public static final class Builder {
 
     private Duration startDelay;
 
+    private StartWorkflowAdditionalOperation startOperation;
```

Comment:
```suggestion
    private StartWorkflowAdditionalOperation additionalOperation;
```

I think this makes more sense as "additional" instead of "start" considering everything on this this builder/options is implied to be for start, but no strong opinion here.

Reply:
Good call 👍